tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -254,11 +254,16 @@ def get_config(self):
         base_config = super().get_config()
         return {**base_config, **config}
 
-    def reset_states(self):
+    def reset_state(self):
         """Resets all of the metric state variables."""
 
         for v in self.variables:
             K.set_value(
                 v,
                 np.zeros((self.num_classes, self.num_classes), v.dtype.as_numpy_dtype),
             )
+
+    def reset_states(self):
+        # Backwards compatibility alias of `reset_state`. New classes should
```

Comment:
Can you add the version `< <TF version>` in the comment? It will be a good remainder to know when we could remove it.

Reply:
Resolved in 9d8e660